### PR TITLE
Created Ruby on Rails - React.js template.

### PR DESCRIPTION
**Pull request by Alejandroq12**
Hello,

Below is the pull request based on the specified project requirements:

- **Initialize Rails Application**: 
  - Created a new Rails app named 'hello-rails-react' using the command `rails new -j webpack hello-rails-react`.
  
- **Gemfile Update**: 
  - Verified and ensured that the `jsbundling-rails` gem is included in the Gemfile.
  
- **Git Initialization**:
  - Initialized the project repository with Git to track changes and versions.
  
- **Node Dependencies**:
  - Installed the necessary node dependencies as outlined in the previous lesson.
  
- **Babel Configuration**:
  - Created a `babel.config.js` in the root folder as per the instructions from the last tutorial.
  
- **Webpack Configuration**:
  - Updated the `webpack.config.js` to incorporate the babel loader, following the guidelines provided in the tutorial.
  
- **Database Setup**:
  - Configured the project to use a PostgreSQL database and established the initial database connection.
  
- **Message Table Creation**:
  - Created a new table specifically for storing messages and added 5 unique greetings.
  
- **API Endpoint Implementation**:
  - Developed an API endpoint that fetches a random greeting from the message table by setting up a new controller action and corresponding Rails route.
  
- **Static View Setup**:
  - Set up a static view to serve as the root for the application.

- **React Integration**:
  - Implemented the `App` component with react-router and integrated it into the static view of the Rails application.
  
- **Greeting Component**:
  - Created the `Greeting` component to display the greeting messages. Configured it as a route within the App component.
  
- **Redux Implementation**:
  - Set up the Redux store, crafted an action, and devised a reducer to facilitate communication with the API endpoint for fetching a random greeting.
  
- **Greeting Display**:
  - Modified the `Greeting` component to display the random greeting fetched from the API.

Thank you for reviewing my pull requests!

Best regards,
Julio Quezada.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,200 @@
...
+
+- [ ] **You may use this template to create your own projects.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
```
Comment (Omar-Muhamad) [inline README.md:166]: - [ ] Please add the issues link here to make your README file more professional.

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Alejandroq12 

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (Alejandroq12) [reply README.md]: Thank you for your feedback.

**Review by LionRouge1 — APPROVED** [review]
Hi @Alejandroq12, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**